include/zcbor_common.h

```diff
@@ -530,7 +530,7 @@ static inline size_t zcbor_flags_to_states(size_t num_flags)
 #define ZCBOR_FLAG_STATES(n_flags) zcbor_flags_to_states(n_flags)
```

Comment:
The changes to zcbor.py is using ZCBOR_FLAG_STATES() in the states[] array size.  The IAR C Compiler does not allow inline methods in array size declarations.   I was able to work around this by expanding the ZCBOR_FLAG_STATES()  macro and not calling zcbor_flags_to_states() (which is then unused and can be removed) .  Not sure if that is the best solution, but seems to be working:

```
#define ZCBOR_ROUND_UP(x, align) (((x) + (align) - 1) / (align) * (align))
#define ZCBOR_FLAG_STATES(num_flags) (ZCBOR_ROUND_UP(num_flags, sizeof(zcbor_state_t) * ZCBOR_BITS_PER_BYTE) / (sizeof(zcbor_state_t) * ZCBOR_BITS_PER_BYTE))

```

Reply:
Thanks for the heads up. Fixed in #456.